Makefile

```diff
@@ -4,6 +4,7 @@ BINARY:=coredns
 SYSTEM:=
 CHECKS:=check godeps
 VERBOSE:=-v
+DOCKER_IMAGE_NAME:=coredns/coredns
```

Comment:
If its for local dev, you shouldn't name it like this, a simple docker push overwrites it on docker hub.

Also: I thought we had (some) agreement of pushing this into the deployment repo.
IOW: why not use a shell alias/function to do this?